utils/colors.ts

```diff
@@ -0,0 +1,53 @@
+/*
+  Note: These color utilities are only used for the demo site.
+  They are not needed for useMqlQuery
+*/
+export const defaultColor = "#3700b3";
+const colors = ["#c862e8", "#7d3Ac1", "#bc5090", "#ea7369", "#f3a533"];
+
+function lightenDarkenColor(col: string, amt: number) {
+  let usePound = false;
+
+  if (col[0] === "#") {
+    col = col.slice(1);
+    usePound = true;
+  }
+
+  const num = parseInt(col, 16);
+
+  let r = (num >> 16) + amt;
+
+  if (r > 255) r = 255;
+  else if (r < 0) r = 0;
+
+  let b = ((num >> 8) & 0x00ff) + amt;
+
+  if (b > 255) b = 255;
+  else if (b < 0) b = 0;
+
+  let g = (num & 0x0000ff) + amt;
+
+  if (g > 255) g = 255;
+  else if (g < 0) g = 0;
+
+  return (usePound ? "#" : "") + (g | (b << 8) | (r << 16)).toString(16);
+}
+
+export default lightenDarkenColor;
+
+export const generateColors = (): string[] => {
```

Comment:
I wouldn't want to make this public as it's a bit fragile.  The colors quickly turn into black or white.  I think we can generate a max of 25ish colors.

Reply:
For future explorers, see https://github.com/transform-data/transform-react/pull/4#discussion_r600929693